tests/integration/test_export_and_heudiconv.py

```diff
@@ -67,100 +68,104 @@ def test_dicom_export():
     for d in subdirs:
         for f in os.listdir(d):
             idx = series_idx(f)
-            assert idx == "9"
+            assert idx == "10"
 
     # ***************************************************************************
     # Test that default overwrite flag is NOT wiping the xnat-export directory
     # Here we test using typer's CLIRunner
     # ***************************************************************************
 
-    cmd = f"{session} {bids_root_dir} -u {xnat_user} -p {xnat_pass} -i 7 -v"
+    cmd = f"{session} {bids_root_dir} -u {xnat_user} -p {xnat_pass} -i 8 -v"
 
     split_cmd = shlex.split(cmd)
 
     r = runner.invoke(export_app, split_cmd)
     print(r.stdout)
 
-    filepath = glob.glob(f"tests/xnat2bids/*/study-*/xnat-export/sub-*/ses-{session_suffix}")[0]
+    filepath = glob.glob(f"tests/xnat2bids/*/study-*/xnat-export/sub-*/ses-{ss}")[0]
     subdirs = [f.path for f in os.scandir(filepath) if f.is_dir()]
 
     assert len(subdirs) == 2
 
     for d in subdirs:
         for f in os.listdir(d):
             idx = series_idx(f)
-            assert idx in ["7", "9"]
+            assert idx in ["8", "10"]
 
-    # ***************************************************************************
-    # Test that overwrite flag is wiping the xnat-export directory
-    # ***************************************************************************
+    # # ***************************************************************************
+    # # Test that overwrite flag is wiping the xnat-export directory
+    # # ***************************************************************************
 
-    cmd = f"{session} {bids_root_dir} -u {xnat_user} -p {xnat_pass} -i 7 -v --overwrite"
+    cmd = f"{session} {bids_root_dir} -u {xnat_user} -p {xnat_pass} -v -i 10 --overwrite"
 
     split_cmd = shlex.split(cmd)
 
     r = runner.invoke(export_app, split_cmd)
     print(r.stdout)
 
-    filepath = glob.glob(f"tests/xnat2bids/*/study-*/xnat-export/sub-*/ses-{session_suffix}")[0]
+    filepath = glob.glob(f"tests/xnat2bids/*/study-*/xnat-export/sub-*/ses-{ss}")[0]
     subdirs = [f.path for f in os.scandir(filepath) if f.is_dir()]
 
     assert len(subdirs) == 1
 
-    for d in subdirs:
```

Comment:
Was there a reason to remove this check? You kind of want to make sure that after `--overwrite` you are left with only the correct series